### ISSUE TITLE
account. _make_transaction is throwing exception without useful information

### ISSUE THREAD

**Issue by klauskobald** [eth-brownie/brownie#1391]
### Environment information

* `brownie` Version: v1.17.2
* Python Version: 3.10
* OS: osx

### What was wrong?

I am using Kovan. Transaction is not created and the exception gives no clue why:

```
  File "<console>", line 1, in <module>
  File "brownie/network/contract.py", line 1267, in __call__
    return fn(*args)  # type: ignore
  File "brownie/network/contract.py", line 1629, in __call__
    return self.transact(*args)
  File "brownie/network/contract.py", line 1502, in transact
    return tx["from"].transfer(
  File "brownie/network/account.py", line 644, in transfer
    receipt, exc = self._make_transaction(
  File "brownie/network/account.py", line 727, in _make_transaction
    raise VirtualMachineError(e) from None
  File "brownie/exceptions.py", line 121, in __init__
    raise ValueError(str(exc)) from None
ValueError: Gas estimation failed: 'The execution failed due to an exception.'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.
```

the thing is - it worked one time and from then on no more. there is tons of ETH and WETH in the wallet for gas.

### How can it be fixed?

adding more info about the exception would be good to get in the account. _make_transaction function.

ps: is there a way to debug the code? I cannot get viscode to debug brownie? - it's not an executable.

**Comment by gnattishness** [comment]
tldr: try running python 3.9.x for now

I'm not too sure how you were able to install brownie using python 3.10.
I'm not using a a Mac, but installation fails for me.

Are you sure you have `brownie 1.17.2`?
e.g. if you can execute `brownie --version`, I would suspect it returns 1.16.4.
or if that doesn't run, try `pip freeze | grep brownie`
I suspect you'll find that it actually has installed 1.16.4 instead.

From what I understand, brownie's vyper dependency does not currently support python 3.10, but their metadata in older versions is overly permissive (`>= 3.6`), so pip will incorrectly think that `vyper==0.2.16` supports 3.10.

**Comment by gnattishness** [comment]
Refer to #1371 - where you see it actually tries to install 1.16.4
> Using cached eth_brownie-1.16.4-py3-none-any.whl (219 kB)
